bin/sam-translate.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python2
+
+"""Convert SAM templates to CloudFormation templates.
+Known limitations: cannot transform CodeUri pointing at local directory.
```

Comment:
Put a blank line between the summary and this line: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings